### ISSUE TITLE
VS Code hangs with this extension

### ISSUE THREAD

**Issue by Maxim-Mazurok** [Shelex/vscode-cy-helper#7]
**Describe the bug**
When I'm trying to edit `gulpfile.js`, VS Code shows "loading" for version control (git) and doesn't accept input in the editor (I can click and move cursor, but no input is added, and then after "loading" is over - it adds input to wherever the cursor is/was.

**To Reproduce**

1. Open `gulpfile.js`
2. Try to edit a function name in it
3. Observe hang/freeze

**Expected behavior**
No hang/freeze

**Environment (please complete the following information):**

-   Extension version: v2.4.0
-   VS Code version: 1.66.2 (user setup)
Commit: dfd34e8260c270da74b5c2d86d61aee4b6d56977
Date: 2022-04-11T07:46:01.075Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
-   OS: Windows_NT x64 10.0.22000

** Developer Tools Console Output **

```text
log.ts:307  WARN Error: Aborted onWillSaveTextDocument-event after 1750ms
    at mainThreadSaveParticipant.ts:37:18
log.ts:313   ERR Cannot read properties of undefined (reading 'label'): TypeError: Cannot read properties of undefined (reading 'label')
    at Function._inflateSuggestDto (vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1698:9960)
    at vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1698:10291
    at Array.map (<anonymous>)
    at Object.provideCompletionItems (vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1698:10282)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:690:15785
    at async Promise.all (index 4)
    at async m (vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:690:15657)
    at async Promise.all (index 0)
```

```text
[2022-04-13 11:48:30.454] [renderer1] [warning] {}
[2022-04-13 11:48:34.553] [renderer1] [error] Cannot read properties of undefined (reading 'label'): TypeError: Cannot read properties of undefined (reading 'label')
    at Function._inflateSuggestDto (vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1698:9960)
    at vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1698:10291
    at Array.map (<anonymous>)
    at Object.provideCompletionItems (vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1698:10282)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:690:15785
    at async Promise.all (index 4)
    at async m (vscode-file://vscode-app/c:/Users/maxim/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:690:15657)
    at async Promise.all (index 0)
```

**Additional context**

**Comment by Maxim-Mazurok** [comment]
Probably related to #5

**Comment by Shelex** [comment]
Hi @Maxim-Mazurok 
Have released v3.0 were introduced couple of performance improvements.
Could you please check it on your side?

**Comment by Maxim-Mazurok** [comment]
> Hi @Maxim-Mazurok 
> Have released v3.0 were introduced couple of performance improvements.
> Could you please check it on your side?

Hello, I think it was kinda intermittent issue, but sure, I'll give it a try and will report back, thank you!
